src/handbook/product/verticals.md

```diff
@@ -1,17 +1,81 @@
 ---
-navTitle: Market Verticals
+navTitle: Product Categories
 ---
 
-# Product Verticals
+# Product Categories
 
-## Background
-Node-RED is best used for connecting _things_ together, and doing logic or manipulation of the data in that connection. Those things can be wide ranging from Web APIs using HTTP to physical hardware on serial connections.
-FlowForge competes in specific market verticals which allow us to optimize for certain use-cases.
+Product categories refer to specific, identifiable segments within an industry or market that have distinct characteristics and needs. By understanding the categories, we can align our [product strategy](./strategy.md) to meet those specific needs. For this overview, we will focus on three distinct product categories: Industrial IoT Platforms, Low-Code Tools, and Data Integration Tools.
 
+## Industrial IoT Platforms
 
-## Digital Integration 
-Also known as IPaaS (Integration Platform as a Service) This market serves business users looking to integrate workflows across multiple different systems. Typically these are Cloud Hosted SaaS types of systems, usually with some sort of Public API.
-There is also the scenario of integrating between cloud systems and internal company systems or data stores.
+An IIoT platform is a comprehensive solution that securely collects, integrates, and manages vast volumes of data from IoT endpoints. It supports various manufacturer and industry protocols, normalizing and analyzing data at the edge and in the cloud. By bridging IT and OT endpoints, it enhances asset management and promotes application development. It can supplement traditional OT functions and pivot asset-intensive industries like manufacturing or energy towards digitalization. The IIoT platform should fulfill key requirements like security, automation, and sustainability. It enables interoperability with various business applications. Key capabilities include device management, integration, data management, analytics, application enablement, and security (see also [IIoT platform definition](https://www.gartner.com/doc/reprints?id=1-2C757S9J&ct=230105&st=sb)). With its prowess in connecting and processing data from IIoT devices, Node-RED finds a fitting place in the sphere of IIoT platforms, becoming an integral tool in industries like manufacturing where the utility of such data is paramount for fostering operational efficacy and stimulating innovation. Further augmenting Node-RED's proficiency, FlowForge introduces enterprise-grade features that streamline the management of remote deployments and delivery pipelines, thus amplifying Node-RED's and FlowForge's potential as IIoT platform.
+
+However, a potential downside is that it may currently be perceived as too general-purpose and not specialized enough for specific IIoT tasks compared to other, more narrowly focused IIoT platforms.
+
```

Comment:
I think the other big risk is that Node-RED is viewed as a developer tool not a platform.